2025.html

```diff
@@ -82,6 +82,7 @@ <h3>Kudos to these websites who <!-- TODO: got --> plan to get naked in 2025!</h
             <li><a href="https://knowler.dev/">Nathan Knowler</a></li>
             <li><a href="https://clarale.com/">Clara Le</a></li>
             <li><a href="https://lume.land/">Lume (a static site generator)</a>
+            <li><a href="https://maribel.dev">maribel.dev</a><li>
```

Comment:
Minor: Please add a trailing slash to the URL. Thank you!

Reply:
applied in https://github.com/css-naked-day/css-naked-day.github.io/pull/234/commits/771b0ed3b7f212efb4dc4d3e89898d533d6d299a.